onnxruntime/core/optimizer/layer_norm_fusion.cc

```diff
@@ -271,9 +271,11 @@ Status LayerNormFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level,
     // remove all the other nodes.
     graph_utils::FinalizeNodeFusion(graph, nodes_to_remove, layer_norm_node);
 
+#ifdef ENABLE_TRAINING
     // add two extra output defs, so we have 3 output defs that match what gradient builder expected
```

Comment:
Where do we disable SkipLayerNormalization if it is training?

Reply:
will check with Sherlock if it's needed, but for now I'll leave it as is.